langchain/document_loaders/mp3_files.py

```diff
@@ -0,0 +1,18 @@
+"""Loader that loads MP3 files."""
+from typing import List
+
+from langchain.audio_models.base import AudioBase
+from langchain.docstore.document import Document
+from langchain.document_loaders.unstructured import UnstructuredFileLoader
+
+
+class MP3Loader(UnstructuredFileLoader):
```

Comment:
should not inherit from UnstructuredFileLoader - Unstructured actually refers to the specific `unstructured` python library, which isnt used... confusing name :/